lib/PuppeteerSharp/Constants.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace PuppeteerSharp
+{
+    /// <summary>
+    /// Constants common to devtools API interaction
+    /// </summary>
+    internal class Constants
```

Comment:
Let's move this to Messaging. 
Let's call it `MessageKeys`.
Let's name constants `CamelCase`.